webpack.config.prod.js

```diff
@@ -65,23 +65,84 @@ export default {
       noInfo: true, // set to false to see a list of every file being bundled.
       options: {
         sassLoader: {
-          includePaths: [path.resolve(__dirname, 'src', 'scss')]
+          includePaths: [path.resolve(__dirname, 'src', 'scss')],
         },
         context: '/',
-        postcss: () => [autoprefixer],
-      }
-    })
+      },
+    }),
   ],
   module: {
     rules: [
-      {test: /\.jsx?$/, exclude: /node_modules/, loader: 'babel-loader'},
-      {test: /\.eot(\?v=\d+.\d+.\d+)?$/, loader: 'url-loader?name=[name].[ext]'},
-      {test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/, loader: 'url-loader?limit=10000&mimetype=application/font-woff&name=[name].[ext]'},
-      {test: /\.[ot]tf(\?v=\d+.\d+.\d+)?$/, loader: 'url-loader?limit=10000&mimetype=application/octet-stream&name=[name].[ext]'},
-      {test: /\.svg(\?v=\d+.\d+.\d+)?$/, loader: 'url-loader?limit=10000&mimetype=image/svg+xml&name=[name].[ext]'},
-      {test: /\.(jpe?g|png|gif)$/i, loader: 'file-loader?name=[name].[ext]'},
-      {test: /\.ico$/, loader: 'file-loader?name=[name].[ext]'},
-      {test: /(\.css|\.scss|\.sass)$/, loader: ExtractTextPlugin.extract('css-loader?sourceMap!postcss-loader!sass-loader?sourceMap')}
-    ]
-  }
+      { test: /\.jsx?$/, exclude: /node_modules/, use: ['babel-loader'] },
+      {
+        test: /\.eot(\?v=\d+.\d+.\d+)?$/,
+        // this is different compared dev, we should review
+        use: [{ loader: 'url-loader', options: { name: '[name].[ext]' } }],
```

Comment:
As stated in the comment, we are using `url-loader` here, but in `dev` we are using `file-loader`.